shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -192,4 +207,16 @@ private void processingComplete(Class<?> entity) {
     private boolean isSuccessfulResponse(HttpResponse<?> httpResponse) {
         return httpResponse.statusCode() >= 200 && httpResponse.statusCode() < 400;
     }
+
+    private void updateManagerRequestMetricsOnSuccess(ManagerRequestType requestType, HttpResponse<Buffer> successResponse) {
+        metricsService.updateManagerRequestMetrics(requestType, ManagerRequestStatus.SUCCESS, String.valueOf(successResponse.statusCode()));
+    }
+
+    private void updateManagerRequestMetricsOnFailure(ManagerRequestType requestType, Throwable error) {
+        if (error instanceof HTTPResponseException) {
```

Comment:
@vaibhavjainwiz If the Error is a `HTTPResponseException`, we'll log the metric twice. Perhaps something like this instead:

```
String statusCode = null;
if(error instanceof HTTPResponseException) {
  statusCode = String.valueOf(((HTTPResponseException) error).getStatusCode());
}
metricsService.updateManagerRequestMetrics(requestType, ManagerRequestStatus.FAILURE, statusCode);

Reply:
sorry, I miss that condition. Thanks for pointing it out.

Reply:
done